ui-admin/src/forms/designer/NewQuestionForm.tsx

```diff
@@ -16,9 +18,10 @@ type NewQuestionFormProps = {
 /** UI for creating a new question. */
 export const NewQuestionForm = (props: NewQuestionFormProps) => {
   const { onCreate, readOnly } = props
-  const [selectedQuestionType, setSelectedQuestionType] = useState<QuestionType>('text')
 
-  const [question, setQuestion] = useState<Question>(baseQuestions[selectedQuestionType])
+  const [question, setQuestion] = useState<Question>(baseQuestions['text'])
+  const [selectedQuestionType, setSelectedQuestionType] = useState<QuestionType | undefined>(undefined)
+  const [freetext, setFreetext] = useState<string | undefined>(undefined)
```

Comment:
It's a matter of style preference, but React's types have a shorthand for this. `useState<T>()` is equivalent to `useState<T | undefined>(undefined)`.

```suggestion
  const [selectedQuestionType, setSelectedQuestionType] = useState<QuestionType>()
  const [freetext, setFreetext] = useState<string>()
```